src/middlewared/middlewared/etc_files/local/ssh/sshd_config.mako

```diff
@@ -48,6 +48,17 @@
 	users = middleware.call_sync('user.query')
 	root_user = filter_list(users, [['username', '=', 'root']], {'get': True})
 
+	login_banner = middleware.call_sync('system.advanced.login_banner')
+	if login_banner != '':
+		with open('/etc/login_banner', 'w', encoding='utf-8') as f:
```

Comment:
We should not be creating multiple files in a single mako file. Create a separate mako for `/etc/login_banner`